draft-ietf-quic-recovery.md

```diff
@@ -518,7 +518,7 @@ or when Initial or Handshake keys are discarded. This ensures the PTO is always
 set based on the latest RTT information and for the last sent packet in the
 correct packet number space.
 
-When ack-eliciting packets in multiple packet number spaces are in flight,
+When ack-eliciting packets in multiple packet number spaces are in-flight,
```

Comment:
I think "in flight" is more correct here.

Reply:
I might be too strict but "in flight" is not defined anywhere. When I looked into stuff relating to "in-flight", I actually had to search both "in-flight" and "in flight". It was inconvenience.

Reply:
This is a quirk in English.  The hyphen is only used when "in-flight" is a modifier for some other term, like "in-flight packets".  https://en.wikipedia.org/wiki/English_compound#Hyphenated_compound_modifiers